### DISCUSSION TITLE
.toolTip()

### DISCUSSION THREAD

**Discussion by Brutalbeard** [RingCentral-Pro-Services/psi-validator.js#11] [Q&A] [answered]
Should we pull these out? Original idea was to get a tool tip method on each validator so the tool tip would be the same across all apps. 

However, thinking about this further, I'm not sure we should become so reliant on that for all our apps. Keep this down to just the validation fun.

**Comment by Brutalbeard** [comment, marked as answer]
@pachori-ujjwal @Brutalbeard @kcleveland 

Thoughts on the above?

**Reply by pachori-ujjwal** [reply]
I think we should pull it out as there is a possibility of having the same data passed through multiple validations while having a simpler tooltip shown to user in order to not scare them.

**Reply by Brutalbeard** [reply]
I also was thinking that if we tie tool tips to this repo, it sort of muddies the water for the scope of the repo. Gonna set up an issue to pull those out and assign to @sagearslan. Will be a good issue to get used to the repo for him.
